.eslintrc.js

```diff
@@ -64,4 +65,15 @@ module.exports = {
     quotes: [ 'error', 'single', { allowTemplateLiterals: true } ],
     semi: [ 'error', 'always' ]
   },
+  overrides:[
```

Comment:
I'm not convinced by these overrides.

I appreciate that you've called them out in the PR description, but I'm not inclined to have different rules/overrides in different packages of the same workspace.

I'd much rather make changes to the code of the files so that they lint successfully.

Please remove the overrides and propose changes to the code so we can continue the discussion on those.

Reply:
@dreamorosi  thanks for the guidance . 

Mh, i make the following question to minimize the assumption because i read in the comment the following:

> but I'm not inclined to have different rules/overrides in different packages of the same workspace.
> I'd much rather make changes to the code of the files so that they lint successfully.

The docs/snippets code may have different requirements because is code written for documentation purposes.
So we do not speak about different rules in different packages. 
We speak about introducing overrides as a subset of exceptional rules. The set of rules of all packages and the subset of rules apply to a specific package docs/snippets. 
This is an invariant and by introducing overrides (subset of rules) as a good practice of flat eslint design approach [link to blog article, eslint lessons learned](https://eslint.org/blog/2022/08/new-config-system-part-1/) , the eslint configuration for code snippets can scale independently.  

What do you think ? Remove the overrides ? If yes, i will do it and we know the option we buy.

Reply:
You are right, my wording wasn't 100% accurate and indeed the purposes for the two types of code are different, so there might be a case in the future in which we make overrides or other types of trade-off (i.e. if we introduce JS code etc).

At the moment however I'd like to prioritize making PR as atomic as possible. By this I mean that I think it's better to focus on one type of change at the time, which for now it's applying the existing linting rules to the new files, without overrides.

If and when we'll consider introducing JS code examples, we'll discuss any change/override needed.

Reply:
@dreamorosi  ok . i do it in this way as proposed, by removing overrides section.